src/tests/test_main.py

```diff
@@ -256,7 +269,7 @@ def tod(d):
                 "uri": main.cdnlink("00666", "elife-00666-video4-data1.xlsx"),
                 "filename": "elife-00666-video4-data1.xlsx"
             }],
-            "image": "https://static-movie-usa.glencoesoftware.com/jpg/10.7554/657/f42a609b0e61fc41798dcba3cc0c87598bd2cf9f/elife-00666-video1.jpg",
+            "image": main.iiiflink("00666", "elife-00666-video1.jpg"),
```

Comment:
Old property is `uri`, new `image` property is a IIIF endpoint so when Journal is changed it will turn it into a full image URI.